edit.js

```diff
@@ -1,126 +1,121 @@
-var dipendenti=[
-    { 
-        nome: "mario rossi",
-        competenzeAssociate: [
+const userList = [
+    {
+        name: "mario rossi",
+        competenceUser: [
             {
-                nome: 'java',
+                name: 'java',
                 area: 'linguaggi programmazione',
-                livello: 'base'
+                level: 'base'
             },
             {
-                nome: 'Linux',
+                name: 'Linux',
                 area: 'Sistemi operativi',
-                livello: 'intermedio'
+                level: 'intermedio'
             }
         ]
-        },
-        { 
-        nome: "franco verdi",
-        competenzeAssociate: [
+    },
+    {
+        name: "franco verdi",
+        competenceUser: [
             {
-                nome: 'js',
+                name: 'js',
                 area: 'linguaggi programmazione',
-                livello: 'avanzato'
+                level: 'avanzato'
             },
             {
-                nome: 'Windows',
+                name: 'Windows',
                 area: 'Sistemi operativi',
-                livello: 'base'
+                level: 'base'
             }
         ]
     }
 ]
-var tempLivelli=['base', 'intermedio', 'avanzato']
 
-for(let i=0;i<dipendenti.length;i++){
-    document.getElementById("employee").innerHTML+='<option value='+i+'>'+dipendenti[i].nome+'</option>'
+
+for (let i = 0; i < userList.length; i++) {
+    const { name } = userList[i];
+    document.getElementById("employee").innerHTML+=`<option value="${i}"> ${ name } </option>`
```

Comment:
spazietti e semicolon sono importanti

```suggestion
    document.getElementById("employee").innerHTML += `<option value="${i}"> ${ name } </option>`;
```